gulpfile.js

```diff
@@ -5,7 +5,7 @@ const { src, dest, parallel, watch, series } = require("gulp"),
   browserSync = require("browser-sync").create();
 
 const FilesPath = {
-  scssFiles: "src/sass/*.scss",
+  scssFiles: "src/sass/index.scss",
```

Comment:
```suggestion
  scssFiles: "src/sass/**/*.scss",
```
Em có thể chuyển như này để nó watch toàn bộ file scss trong folder /src/sass/ nhé